inc/config.class.php

```diff
@@ -167,7 +166,7 @@ public static function preConfigSet(Config $config): void
             // Even if config already exists, it is submitted on `inventory` context and will therefore be
             // considered as new. We have to call `Config::setConfigurationValues()` using the good context to be able to
             // trigger `add` or `update` whether config already exists or not.
-            $config->input = false;
+            $config->input = [];
```

Comment:
Be careful with that kind of change. `CommonDBTM::input` can be `false`; there are several places where code checks if it's an array or not. This kind of conversion may lend to unexpected behavior.